src/sentry/statistical_detectors/issue_platform_adapter.py

```diff
@@ -0,0 +1,74 @@
+import hashlib
+import uuid
+from datetime import datetime, timezone
+from typing import List
+
+from sentry.issues.grouptype import PerformanceDurationRegressionGroupType
+from sentry.issues.issue_occurrence import IssueEvidence, IssueOccurrence
+from sentry.issues.producer import produce_occurrence_to_kafka
+from sentry.seer.utils import BreakpointData
+from sentry.utils import metrics
+
+
+def fingerprint_regression(transaction, automatic_detection=True):
+    prehashed_fingerprint = (
+        f"p95_transaction_duration_regression-{transaction}"
+        if automatic_detection
+        else f"p95_transaction_duration_regression-{transaction}-experiment"
+    )
+    return hashlib.sha1((prehashed_fingerprint).encode()).hexdigest()
+
+
+class Regression(BreakpointData):
+    project_id: int
```

Comment:
`BreakpointData` unfortunately doesn't contain `project_id` that we need for `IssueOccurrence`

Reply:
`BreakpointData` contains the `project` key which contains the project id as a str. A little unfortunate but it's how the response is formatted. Though we could massage it into something more usable once we receive the response instead of introducing another key.

Reply:
for transactions, we've been passing project slug instead of project ids 😬 because we had a frontend dependency on project slug being in the response

Reply:
I can refactor the frontend so that it doesn't break when we're passing a project id 🤔

Reply:
I would advise against using the slug for this. The queries are now cross orgs so in the unlikely event that 2 orgs use the same project slug, then we're going to have a sneaky bug to hunt down.

Reply:
oh I see what you're saying. Ok yeah I'm going to refactor the frontend to accept project ids and not slugs 👍